amm/drink-tests/src/stable_swap_tests/tests_getters.rs

```diff
@@ -0,0 +1,129 @@
+use super::*;
+
+// ref https://github.com/ref-finance/ref-contracts/blob/d241d7aeaa6250937b160d56e5c4b5b48d9d97f7/ref-exchange/tests/test_stable_pool.rs#L23
+#[drink::test]
+fn test_01(mut session: Session) {
+    let initial_reserves = vec![100000 * ONE_DAI, 100000 * ONE_USDT, 100000 * ONE_USDC];
+    let (stable_swap, tokens) = setup_stable_swap_with_tokens(
+        &mut session,
+        vec![18, 6, 6],
+        initial_reserves.iter().map(|amount| amount * 10).collect(),
+        10_000,
+        25,
+        2000,
+        BOB,
+    );
+
+    _ = stable_swap::add_liquidity(
+        &mut session,
+        stable_swap,
+        BOB,
+        1,
+        initial_reserves.clone(),
+        bob(),
+    )
+    .expect("Should successfully add liquidity");
+
+    assert_eq!(
+        stable_swap::tokens(&mut session, stable_swap),
+        tokens,
+        "Incorrect token accounts"
+    );
+    assert_eq!(
+        stable_swap::reserves(&mut session, stable_swap),
+        initial_reserves,
+        "Incorrect reserves"
+    );
+    assert_eq!(
+        stable_swap::amp_coef(&mut session, stable_swap),
+        10_000,
+        "Incorrect A"
+    );
+    assert_eq!(
+        stable_swap::fees(&mut session, stable_swap),
+        (25, 2000),
```

Comment:
You could extract fees and amp_coef to vars and use here and in the constructor.